docs/rules/index.md

```diff
@@ -4,109 +4,111 @@ sidebarDepth: 0
-| [regexp/no-empty-capturing-group](./no-empty-capturing-group.md) | disallow capturing group that captures empty. | :star: |
-| [regexp/no-empty-character-class](./no-empty-character-class.md) | disallow character classes that match no characters |  |
-| [regexp/no-empty-group](./no-empty-group.md) | disallow empty group | :star: |
-| [regexp/no-empty-lookarounds-assertion](./no-empty-lookarounds-assertion.md) | disallow empty lookahead assertion or empty lookbehind assertion | :star: |
-| [regexp/no-escape-backspace](./no-escape-backspace.md) | disallow escape backspace (`[\b]`) | :star: |
-| [regexp/no-invalid-regexp](./no-invalid-regexp.md) | disallow invalid regular expression strings in `RegExp` constructors | :star: |
-| [regexp/no-lazy-ends](./no-lazy-ends.md) | disallow lazy quantifiers at the end of an expression | :star: |
-| [regexp/no-misleading-unicode-character](./no-misleading-unicode-character.md) | disallow multi-code-point characters in character classes and quantifiers | :wrench: |
-| [regexp/no-missing-g-flag](./no-missing-g-flag.md) | disallow missing `g` flag in patterns used in `String#matchAll` and `String#replaceAll` | :wrench: |
-| [regexp/no-optional-assertion](./no-optional-assertion.md) | disallow optional assertions | :star: |
-| [regexp/no-potentially-useless-backreference](./no-potentially-useless-backreference.md) | disallow backreferences that reference a group that might not be matched | :star: |
-| [regexp/no-super-linear-backtracking](./no-super-linear-backtracking.md) | disallow exponential and polynomial backtracking | :star::wrench: |
-| [regexp/no-super-linear-move](./no-super-linear-move.md) | disallow quantifiers that cause quadratic moves |  |
-| [regexp/no-useless-assertions](./no-useless-assertions.md) | disallow assertions that are known to always accept (or reject) | :star: |
-| [regexp/no-useless-backreference](./no-useless-backreference.md) | disallow useless backreferences in regular expressions | :star: |
-| [regexp/no-useless-dollar-replacements](./no-useless-dollar-replacements.md) | disallow useless `$` replacements in replacement string | :star: |
-| [regexp/strict](./strict.md) | disallow not strictly valid regular expressions | :star::wrench: |
+| Name                                                                                             | Description                                                                             | 💼 | ⚠️ | 🔧 | 💡 | ❌  |
+| :----------------------------------------------------------------------------------------------- | :-------------------------------------------------------------------------------------- | :- | :- | :- | :- | :- |
+| [no-contradiction-with-assertion](../../docs/rules/no-contradiction-with-assertion.md)           | disallow elements that contradict assertions                                            |    |    |    | 💡 |    |
```

Comment:
As mentioned [here](https://github.com/ota-meshi/eslint-plugin-regexp/pull/493#discussion_r1057349245), these relative links are broken. The problem is that vitepress uses `docs/` as the root directory.

To fix this, please optimize/shorten relative paths.

Reply:
Good catch, will attempt to fix as part of https://github.com/bmish/eslint-doc-generator/issues/284.

Reply:
This should be fixed now: https://github.com/bmish/eslint-doc-generator/releases/tag/v1.4.1

Please verify that the rule links look correct.

Reply:
Links work perfectly now. Thank you!